src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSliderAutomationPeer.cs

```diff
@@ -91,51 +90,28 @@ public void RaisePropertyChangedEvent(Color oldColor, Color newColor, int oldVal
 
 		private string GetValueString(Color color, int value)
 		{
-			if (DownlevelHelper.ToDisplayNameExists())
+			string resourceStringWithName;
+			switch ((Owner as ColorPickerSlider).ColorChannel)
 			{
-				string resourceStringWithName;
-				switch (_owner.ColorChannel)
-				{
-					case ColorPickerHsvChannel.Hue:
-						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringHueSliderWithColorName);
-						break;
-					case ColorPickerHsvChannel.Saturation:
-						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringSaturationSliderWithColorName);
-						break;
-					case ColorPickerHsvChannel.Value:
-						resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringValueSliderWithColorName);
-						break;
-					default:
-						return string.Empty;
-				}
-
-				return StringUtil.FormatString(
-					resourceStringWithName,
-					value,
-					ColorHelper.ToDisplayName(color));
+				case ColorPickerHsvChannel.Hue:
+					resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringHueSliderWithColorName);
+					break;
+				case ColorPickerHsvChannel.Saturation:
+					resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringSaturationSliderWithColorName);
+					break;
+				case ColorPickerHsvChannel.Value:
+					resourceStringWithName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringValueSliderWithColorName);
+					break;
+				default:
+					return string.Empty;
 			}
-			else
-			{
-				string resourceStringWithoutName;
-				switch (_owner.ColorChannel)
-				{
-					case ColorPickerHsvChannel.Hue:
-						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringHueSliderWithoutColorName);
-						break;
-					case ColorPickerHsvChannel.Saturation:
-						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringSaturationSliderWithoutColorName);
-						break;
-					case ColorPickerHsvChannel.Value:
-						resourceStringWithoutName = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ValueStringValueSliderWithoutColorName);
-						break;
-					default:
-						return string.Empty;
-				}
 
-				return StringUtil.FormatString(
-					resourceStringWithoutName,
-					value);
```

Comment:
Seems like a big change, was it removed from the original source?

Reply:
Yes, anything with `DownlevelHelper` has been removed in WinUI3.

Reply:
It makes sense to remove some of these helpers as WinUI3 was a new jump in the minimum supported feature set. I ported the DownlevelHelper originally because it was used quite a bit in the code base. I also used it to disable .ToDisplayName(), etc. because the algorithm wasn't open sourced at the time to calculate that. I see you've added that algorithm now as well so there doesn't seem to be a reason to keep DownlevelHelper for color picker at least.